docs/ssh-key-pair-setup.md

```diff
@@ -0,0 +1,18 @@
+# SSH Key Pair Setup
+
+## Creating a SSH key pair on Windows machines
+- Create a key pair using PuTTYgen. Please refer to https://www.ssh.com/ssh/putty/windows/puttygen for instructions to create SSH keys. 
+- Once the public / private key pair is created, the SSH2 public key format must be converted to a OpenSSH format for Terraform. Please refer to https://tutorialinux.com/convert-ssh2-openssh/ for detailed instructions. 
```

Comment:
we can put the reference after our instructions in case they need to know more, so this section is:
1. download puttygen from ...
2. create sshkey by clicking "Generate" with "RSA" selected
3. run the following command in command prompt to convert ....
'command to run'
For details, refer to
- link to puttygen
- link to convert key